rplugin/python3/deoplete/sources/deoplete_jedi/server.py

```diff
@@ -210,6 +210,8 @@ def _loop(self):
 
             if not out and cache_key[-1] in ('package', 'local'):
                 # The backup plan
+                # TODO(blueyed): remove this (far too less results for e.g.
+                # numpy), or at least do not cache it to disk.
```

Comment:
Indicating to not cache it is not possible currently, but could be done later when the Server part is removed.